ENIGMAsystem/SHELL/Platforms/General/PFwindow.h

```diff
@@ -237,6 +237,11 @@ int window_get_region_height();
 int window_get_region_width_scaled();
 int window_get_region_height_scaled();
 
+void window_set_min_width(int width);
+void window_set_min_height(int height);
+void window_set_max_width(int width);
+void window_set_max_height(int height);
```

Comment:
Maybe we want to move these to General so that the game has them for all platfoms. We're going to want to add this to Win32 anyway with WM_GETMINMAXINFO message. I think we'll need the state there anyway, not sure.